### PR TITLE
CMA add specific parameters for OCP Prometheus

### PR DESCRIPTION
**Pull request by mburke5678**
Per [Slack](https://redhat-internal.slack.com/archives/C02F1J9UJJD/p1687329664428689?thread_ts=1687289719.175849&cid=C02F1J9UJJD), add the following:

```
You can now deploy a scaled object or scaled job to enable autoscaling for your application, as described in the following sections. 
To use OpenShift Container Platform monitoring as the source, in the trigger you need to specify:
 - `prometheus` trigger type
 -  `https://thanos-querier.openshift-monitoring.svc.cluster.local:9092` as the `serverAddress`
 - `authModes` needs to be set to `bearer`
 - `namespace` needs to be set to the namespace of the object you want to scale
 - `authenticationRef` needs to point to the trigger authentication resource specified in step 4
```

### CODE REVIEW

**Review by zroubalik — APPROVED** [review]
LGTM

Thanks for the quick fix @mburke5678 !

**Comment by GroceryBoyJr** [comment]
Found the legit preview link: https://61509--docspreview.netlify.app/openshift-enterprise/latest/nodes/cma/nodes-cma-autoscaling-custom-trigger.html#nodes-cma-autoscaling-custom-prometheus-config_nodes-cma-autoscaling-custom-trigger

**Comment by GroceryBoyJr** [comment]
@mburke5678 This update looks like it would also go well in a release note.

**Comment by GroceryBoyJr** [comment]
@mburke5678 The section you refer to,: "Understanding how to add custom metrics autoscalers" only exists in the 4.13 documentation but you seek to apply this update to all sections from 4.10+. Are you sure it's applicable to 4.10, 4.11 and 4.12?

**Review by GroceryBoyJr — COMMENTED** [review]
@mburke5678 only one REALLY tiny nit, sir. Excellent work.

File: modules/nodes-cma-autoscaling-custom-prometheus-config.adoc [hunk 1]
```diff
@@ -173,5 +173,11 @@ subjects:
 $ oc create -f <file-name>.yaml
 ----
 
-You can now deploy a scaled object or scaled job to enable autoscaling for your application, as described in the following sections. To use {product-title} monitoring as the source, in the trigger, or scaler, specify the `prometheus` type and use `\https://thanos-querier.openshift-monitoring.svc.cluster.local:9092` as the `serverAddress`.
+You can now deploy a scaled object or scaled job to enable autoscaling for your application, as described in "Understanding how to add custom metrics autoscalers." To use {product-title} monitoring as the source, in the trigger, or scaler, you must include the following parameters:
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/nodes-cma-autoscaling-custom-prometheus-config.adoc:176]: @GroceryBoyJr Is this a new thing? I've always put the period inside the end quote.

**Pull request merged**